gemrb/GemRB.cpp

```diff
@@ -32,6 +32,13 @@
 #include <unistd.h>
 #endif
 
+#ifdef __amigaos4__
+#include "exports.h"
+static const char __attribute__((used)) stackcookie[] = "$STACK: 2000000";
+static const char __attribute__((used)) *version      = "$VER: GemRB COVID-19 for AmigaOS4.1 (2020-10-13)";
```

Comment:
Shouldn't this use VERSION_GEMRB or GEMRB_STRING, so it's not always the same?

Reply:
This is an internal stuff amiga. I've changed it locally to test. Now i've added GEMRB_STRING (not yet committed)